src/app/dynamic/enterprise/quotas/quota-widget/template.html

```diff
@@ -54,7 +55,7 @@
       </ng-container>
     </div>
 
-    <mat-card class="km-detail-view"
+    <mat-card class="km-compact-detail-on-hover"
```

Comment:
```suggestion
    <mat-card class="km-quota-detail"
```

Reply:
I think `km-quota-hover-details` will be more appropriate.

Reply:
`hover` should not be in class name. If we want to style something on hover then it should be done using `:hover` selector